build.gradle

```diff
@@ -184,43 +184,6 @@ configure modsRegular,
         new File("$rootDir/tmp/workspace".toString()).mkdirs()
         new File("$rootDir/tmp/logs".toString()).mkdirs()
 
-        // TODO fix / move flaky tests so we can make ignoreFailures false again
-        ignoreFailures = TestTools.allowFailure project
-        if (ignoreFailures) {
-            // begin dirty trick to ensure you can re-run failed tests.
-            // Note that this workaround is only needed because of the ignoreTestFailures=true "footgun" above.
-            File cacheBust = new File(p.buildDir, "testState/${path.replaceAll(":", "_")}".toString())
-            if (!cacheBust.exists()) {
-                cacheBust.parentFile.mkdirs()
-                cacheBust.createNewFile()
-            }
-            if (cacheBust.text != 'passed') {
-                // If the cacheBust file exists and it did not pass on last run, fill it w/ a unique value to force the task to rerun
-                cacheBust.text = "rerun-${UUID.randomUUID()}"
-            }
-            outputs.file(cacheBust)
```

Comment:
I think we're going to still want cache bust in the ide.  You can limit it w/ `if (System.getProperty("idea.version"))`

Reply:
So - with all this logic removed, I'm able to verify that I'm cache-ing green tests on the command line w/ gradle. But from my runs via IJ, it *seems* to run the tests every time.